.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,79 @@
+name: Bug Report
+about: Create a report to help us improve Electron
+title: "[Bug]: "
+body:
+- type: checkboxes
+  attributes:
+    label: Preflight Checklist
+    description: Please ensure you've completed all of the following.
+    options:
+      - label: I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
+        required: true
+      - label: I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
+        required: true
+      - label: I have searched the issue tracker for a feature request that matches the one I want to file, without success.
+        required: true
+- type: input
+  attributes:
+    label: Electron Version
+    description: What version of our software are you running?
+    placeholder: 11.0.0
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: What platform are you using?
+    options:
+      - macOS Catalina
+      - macOS Mojave
+      - macOS Big Sur
+      - Windows 10
+      - Windows 8
+      - Ubuntu
+      - Other Linux
+      - Windows on Arm
+      - Other (specify in a later section)
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Last Known Working Electron version
+    description: What is the last version of Electron this worked in, if applicable?
+    placeholder: 10.0.0
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A clear and concise description of what you expected to happen.
```

```suggestion
    description: A clear description of what should happen.
```

Rationale: more words bad. fewer words good

---

In some ways I prefer "what you expected" because it's a better signal for our own documentation being unclear i think 🤔